docs/run.md

```diff
@@ -128,10 +128,12 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>`, `<private key file>` and [Azure
+Marketplace image](
+https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage) to replace in below command. It may take several minutes to complete.
```

Comment:
seems duplicated here
![image](https://user-images.githubusercontent.com/10083705/115201444-eb651980-a127-11eb-818c-6cd0aef95a81.png)

Reply:
The hyper link may sound a duplicate one, but to have a sync between the command parameter and description mentioning the azure market place image would still be relevant IMO..

Reply:
@LiliDeng It looks the image is not an optional argument, so many people is trying to add it to command. Can you have a look? If it's not easy to fix, please create a task to me.

Reply:
With the latest changes, the Marketplace image is not a required parameter. Please remove this change, and leave only fix on ready.yml.

Reply:
Sure, removed changes related to marketplace_image. Thanks!